### PR TITLE
Remove the settings for tm.manager_hook

### PR DESCRIPTION
**Pull request by stevepiercy**
See https://github.com/mmerickel/pyramid/pull/8#issuecomment-441482617

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py [hunk 1]
```diff
@@ -57,7 +57,6 @@ def includeme(config):
 
     """
     settings = config.get_settings()
-    settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
```
Comment (mmerickel) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py]: do not remove it from the alchemy version, only zodb

**Comment by stevepiercy** [comment]
Upon merge, the ZODB+Traversal wiki tutorial in the Pyramid docs will also need to be updated.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py (hunk 1 above)
Comment (stevepiercy) [reply {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py]: oops. fixed.

**Review by mmerickel — APPROVED** [review]

**Comment by stevepiercy** [comment]
One more thing, there are no other instances of `pyramid_tm` in the zodb option, AFAICT, so should I remove this line as well?

https://github.com/stevepiercy/pyramid-cookiecutter-starter/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/setup.py#L26

**Comment by mmerickel** [comment]
that's not true, and no

**Comment by stevepiercy** [comment]
Ah, ok, found it. Thanks for checking.

**Pull request merged**